unobstructed_lanechange/include/spline.h

```diff
@@ -0,0 +1,404 @@
+/*
+ * spline.h
```

Comment:
Not sure if you have another way to deal with it. But generally we do not want to include third party file in our repo.